fla/models/utils.py

```
@@ -348,16 +351,15 @@ def __len__(self):
     def get_seq_length(self, layer_idx: int | None = 0, cache_position=None) -> int:
```

Ruff (0.15.0)

[warning] Unused method argument: `cache_position`

(ARG002)